bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml

```diff
@@ -323,7 +323,7 @@ limitations under the License.
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
-            <version>${hadoop.version}</version>
```

Comment:
Shouldn't this be set for bigtable-hbase-2.x-integration-tests as well?

Reply:
hmm, let me take a closer look